<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerview</class>
 <widget class="QDialog" name="customerview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>551</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_init_location"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_location"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fare"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_time"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_returntime"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ref"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Book!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Ref</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
